### PR TITLE
adding tests to verify pdf-git option and its error messages

### PR DESCRIPTION
**Pull request by omehes**

### CODE REVIEW

**Comment by brittweinstein** [comment]
@omehes please put points on this

File: frontend/pages/index.vue [hunk 1]
```diff
@@ -41,7 +41,7 @@
                     <v-radio-group v-model="jobType" row mandatory :default="jobTypes.PDF" @change="resetFormValidation">
                       <v-radio label="PDF" :value="jobTypes.PDF" class="pdf-radio-button"></v-radio>
                       <v-radio label="Web Preview" :value="jobTypes.DIST_PREVIEW" class="preview-radio-button"></v-radio>
-                      <v-radio label="PDF (git)" :value="jobTypes.GIT_PDF" class="git-pdf-radio-button"></v-radio>
+                      <v-radio label="PDF (git)" :value="jobTypes.GIT_PDF" class="git-pdf-radio-button" ></v-radio>
```
Comment (tomjw64) [inline frontend/pages/index.vue:44]: unnecessary change

File: frontend/pages/index.vue [hunk 2]
```diff
@@ -51,7 +51,7 @@
                         v-model="collectionId"
                         :rules="activeJobIsUsingArchive ? collectionRules : gitCollectionRules"
                         label="Collection ID"
-                        class="collection-id-error-text collection-id-incorrect-error-text collection-id-field"
+                        class="collection-id-error-text git-pdf-error-text collection-id-incorrect-error-text git-pdf-incorrect-error-text collection-id-field"
```
Comment (tomjw64) [inline frontend/pages/index.vue:54]: additional classes are not needed, we should use the same class for locating the same element, ideally just `collection-id-field` here

**Review by tomjw64 — CHANGES_REQUESTED** [review]

**Comment by omehes** [comment]
@tomjw64 changes you requested are done. ready for another review :)

**Review by tomjw64 — APPROVED** [review]

**Pull request merged**